pyproject.toml

```diff
@@ -20,6 +20,8 @@ dependencies = [
     "numpy>=1.24.1",
     "matplotlib>=3.6.2",
     "typing-extensions>=4.5.0",
+    "ipykernel>=6.22.0",
```

Comment:
Does Pacti really need `ipykernel`?

Reply:
Not it doesn't; I had used it when trying examples using notebooks. I'll remove it.